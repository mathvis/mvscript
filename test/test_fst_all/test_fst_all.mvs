func a(b: [int]) vector {
	let c = 0;
	return Vector(5, 3);
}
