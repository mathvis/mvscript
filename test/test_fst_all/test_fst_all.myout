(Decl (FunctionDef (FunctionIdentifier "a") [(VarIdentifier "b",ArrayT IntT)] VectorT (Just (Block FunctionBlock [Decl (Variable (VarIdentifier "c") (Just IntT) (Just (Type (Int 0))))]))),ParserState {config = Configuration {debug = True, collapseOperations = True, collapseControlFlow = True, colors = True}, st = fromList [("c",VariableData {variableType = Just IntT, inScope = False, isInitialized = True, isConstant = False})], fst = fromList [("a",FunctionData {returnType = VectorT, arguments = [("b",ArrayT IntT)], hasBody = True})]})
