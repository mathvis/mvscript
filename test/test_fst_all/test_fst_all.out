(Decl (FunctionDef (FunctionIdentifier "a") [(VarIdentifier "b",ArrayT IntT)] VectorT (Just (Block (FunctionBlock VectorT) [Decl (Variable (VarIdentifier "c") (Just IntT) (Just (Type (Int 0)))),Expr (Return (Just (Type (Vector [Type (Int 5),Type (Int 3)]))))]))),ParserState {config = Configuration {debug = True, collapseOperations = True, collapseControlFlow = True, colors = True}, st = fromList [("c",VariableData {variableType = Just IntT, inScope = False, isInitialized = True, isConstant = False})], fst = fromList [("a",FunctionData {returnType = VectorT, arguments = [("b",ArrayT IntT)], hasBody = True})], context = Nothing, unresolvedFunctionCalls = []})
